docs/apache-airflow/upgrading-to-2.rst

```diff
@@ -286,10 +286,14 @@ When DAGs are initialized with the ``access_control`` variable set, any usage of
 .. warning::
     Breaking change
 
-    Previously we were using two versions of UI, which were hard to maintain as we need to implement/update the same feature
-    in both versions. With this release we've removed the older UI in favor of Flask App Builder RBAC UI. No need to set the
-    RBAC UI explicitly in the configuration now as this is the only default UI. We did it to avoid
-    the huge maintenance burden of two independent user interfaces
+    Previously we were using two versions of the UI. This was difficult to maintain, because it meant we
```

Comment:
```suggestion
    Previously we were using two versions of the UI:
        - non-RBAC UI
        - Flask App Builder RBAC UI
     This was difficult to maintain, because it meant we
```

Should we say explicitly what were the two versions?